_component_design/switches.md

```diff
@@ -19,29 +18,81 @@ helix-ui-javascript: false
 
 <section class="static-section" markdown="1">
 
-## Introduction
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use a switch in the following situations:
+
+- When you want to enable to user to make a binary decision, such as turning a setting on or off. A switch does not contain a neutral option. Modifying preferences and settings is a prime use case for this component.
+- Use a switch when the user needs to modify settings or enable billing and notification preferences. If your use case is outside of these three areas, consider a [radio]({{site.baseurl}}/components/radio-buttons.html) or [checkbox]({{site.baseurl}}/components/checkboxes.html).
```

Comment:
this section is simultaneously too specific and too vague...maybe try something like this:

When a user needs to make a binary decision, such as turning a setting on or off or setting a preference enabling or disabling a device or system.

If your use case is outside of these areas, consider a [radio]({{site.baseurl}}/components/radio-buttons.html) or [checkbox]({{site.baseurl}}/components/checkboxes.html) or [selector strip] (link to selector strip page)

Reply:
Done.